src/plugins/data/public/search/errors/es_error.tsx

```diff
@@ -17,7 +17,7 @@ export class EsError extends KbnError {
   readonly attributes: IEsError['attributes'];
 
   constructor(protected readonly err: IEsError) {
-    super('EsError');
+    super(`EsError: ${getRootCause(err)?.reason || err.attributes?.reason || 'unknown'}`);
```

Comment:
just interesting is `|| err.attributes?.reason` needed? I see that case is handled in `getRootCause -> getNestedCause.`